documentation/src/main/jbake/content/documentation.adoc

```diff
@@ -16,6 +16,9 @@ Talend Component framework relies on several primitive components.
 They can all use `@PostConstruct` and `@PreDestroy` to initialize/release
 some underlying resource at the beginning/end of the processing.
 
+IMPORTANT: in distributed environments class' constructor will be called on driver node, methods annotated with
```

Comment:
driver/worker is not obvious and only relevant in spark context, maybe adding a schema (you can use asciidoctor-diagram) to represent the workflow (partition plan computation then execution) can help and put a name on each phases, wdyt?